### PR TITLE
docs: add instructions on how to use priority for menu items

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
As per title.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [x] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

File: docs/extension-system/extension-types/app-menu-items.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ A `path` will result in an `<a>` element that links to an internal page via the
 
 A `url` will result in an `<a>` element that links to an external page. External pages always open in a new tab or window.
 
-At least one of these properties has to be provided when registering an extension. If you define more than one, the priority order is `handler` > `path` > `url`. 
+At least one of these properties has to be provided when registering an extension. If you define more than one, the priority order is `handler` > `path` > `url`.
+
+`priority` specifies the order of the menu item. 50 is probably a good place to start, then go up/down based on where the item should be placed. Defaults to the highest possible number, so the item will most likely end up at the bottom of the list.
```
Comment (kulmann) [inline docs/extension-system/extension-types/app-menu-items.md:46]:
```suggestion
`priority` specifies the order of the menu items. 50 is a good number to start with, then go up/down based on where the item should be placed. Defaults to the highest possible number, so the item will most likely end up at the bottom of the list if you don't specify a `priority`.
```

Comment (kulmann) [reply docs/extension-system/extension-types/app-menu-items.md:46]: Do we have a recommendation? Not setting anything? Or trying to define a sane priority?

Comment (JammingBen) [reply docs/extension-system/extension-types/app-menu-items.md:46]: Hard to say, is it valid to not have a recommendation? We explained the effects and how to use it, now it's up to the app creator what to choose I'd say 🤷

Comment (kulmann) [reply docs/extension-system/extension-types/app-menu-items.md:46]: Then maybe `Leave it empty if unsure what to pick`?

**Review by kulmann — APPROVED** [review]

**Pull request merged**